universal/src/logging/impl/formatters/json.cpp

```diff
@@ -40,7 +40,10 @@ void Json::AddTag(std::string_view key, const LogExtra::Value& value) {
     std::visit(
         [&, this](const auto& x) {
             sb_.Key(key);
-            WriteToStream(x, sb_);
+            if constexpr (std::is_same_v<decltype(x), const JsonString&>)
+                sb_.WriteRawString(x.Value());
```

Comment:
Add a `WriteToString` overload after `JsonString` definition, use `json_fwd.hpp` to avoid pulling in unnecessary headers

Reply:
fixed